tpp-app/src/services/utils/constants.js

```diff
@@ -1,18 +1,17 @@
 export const FLOW_LEVEL = {
-    NONE: "NONE",
-    LIGHT: "LIGHT",
-    MEDIUM: "MEDIUM",
-    HEAVY: "HEAVY",
-    SPOTTING: "SPOTTING"
+  NONE: "NONE",
+  LIGHT: "LIGHT",
+  MEDIUM: "MEDIUM",
+  HEAVY: "HEAVY",
+  SPOTTING: "SPOTTING"
 }
 
 export const MOOD_LEVEL = {
   HAPPY: "HAPPY",
   SAD: "SAD",
   NEUTRAL: "NEUTRAL",
   SICK: "SICK",
-  ANGRY: "ANGRY",
-  CUSTOM: "" // empty string
```

Comment:
Are we getting rid of custom? Same goes for the custom in `EXERCISE_TYPE`

Reply:
Maybe @faye-tan and @yichenesia can help clarify here but I recall hearing something at a coworking about no longer using custom emojis for mood and exercise?

Reply:
Wiki has been updated! We are indeed no longer using custom emojis at all, MOOD_LEVEL and EXERCISE should have 9 defined emojis: https://github.com/uoftblueprint/the-period-purse/wiki/Models-and-AsyncStorage-Keys